src/AWS.Deploy.Recipes/CdkTemplates/BlazorWasm/Generated/Recipe.cs

```diff
@@ -132,6 +132,7 @@ private void ConfigureCloudFrontDistribution(Configuration settings)
                     var loggingBucket = new Bucket(this, nameof(AccessLoggingBucket), InvokeCustomizeCDKPropsEvent(nameof(AccessLoggingBucket), this, new BucketProps
                     {
                         RemovalPolicy = RemovalPolicy.RETAIN,
+                        AccessControl = BucketAccessControl.LOG_DELIVERY_WRITE
```

Comment:
Unrelated to the CDK version, noticed that this is failing as well due to a change in S3 behavior last year. See https://github.com/aws/aws-cdk/issues/27571 for more info.